### ISSUE TITLE
nnet training in version 2.0

### ISSUE THREAD

**Issue by hannabonert** [MontrealCorpusTools/Montreal-Forced-Aligner#210]
Hello,
Thank you for this great tool!
I was wondering whether nnet training has been removed in version 2.0?
I don't see a trainer class for it, and there doesn't seem to be an option for it in the parsing of the YAML training config file in the train_yaml_to_config function in config/__init__.py. Version 1.1 does have these, though.
Are there plans to include it in the future?

Thank you very much!

**Comment by mmcauliffe** [comment]
Hi, just to follow up, I did remove it for 2.0 for a couple of reasons.  

The main one is that I'm not too sure that it improves the quality of alignment while adding quite a bit of complexity and training time.  I think more gains in alignment accuracy would probably come first from other areas.  

The second is that the kaldi nnet system doesn't feel particularly production ready, since they have three subtly different setups, and I've seen notes about them wanting to incorporate PyTorch models.  If they do go the PyTorch integration route, I'd be much happier including a neural network acoustic model training regime, since that should be much easier for me to follow than the usual mix of bash/perl/python that the recipes use.

With that said, @a-coles did put in significant effort into figuring out the nnet recipes and implementing them in MFA, so I hesitated to remove them, but it was a bit of a daunting task to go through and refactor them for 2.0, but can maybe think about starting to add that code back in once my other efforts in speaker classification and transcription are implemented, since I think the benefits of DNN acoustic models are much more apparent for transcription than alignment.
